### PR TITLE
class-builder: `elem === ''` means no elem

### PR DESCRIPTION
**Pull request by indutny**
Fix: #152

### CODE REVIEW

**Comment by zxqfox** [comment]
We just though about classBuilder as a https://github.com/bem/bem-naming clone. What you think about using `.stringify` in bem-xjst?

**Comment by indutny** [comment]
I'm fine with this as long as benchmarks are happy with `.stringify`

**Comment by miripiruni** [comment]
IMHO, I would like to merge PR as is. And open another one for [modVal and elemModVal](https://github.com/bem/bem-xjst/commit/6e05992e14340174e23d326831677988d50b7295).

**Comment by zxqfox** [comment]
Yeah, agree.

**Comment by tadatuta** [comment]
lgtm

File: lib/bemxjst/class-builder.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function ClassBuilder(options) {
 exports.ClassBuilder = ClassBuilder;
 
 ClassBuilder.prototype.build = function build(block, elem) {
-  if (elem === undefined)
+  if (!elem)
```
Comment (zxqfox) [inline lib/bemxjst/class-builder.js]: Guess we can add here `elem !== 0` now or later?

**Comment by zxqfox** [comment]
Lgtm for now

**Pull request merged**